File: rtcbuilder/trunk/jp.go.aist.rtm.rtcbuilder.csharp/resource/AIST7/testImpl.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace foo
{
    public class testImpl : MarshalByRefObject, test
    {
        public void _Set(System.Int16[] Data)
        {
            throw new Exception("The method or operation is not implemented.");
        }
        public System.Int32 Set2(System.Int16[] Data)
        {
            throw new Exception("The method or operation is not implemented.");
        }
    }
}

File: RELENG_1_0_eclipse32/rtmtools/jp.go.aist.rtm.rtcbuilder.vbdotnet/resource/AIST7/testImpl.vb
Public Class testImpl
    Inherits MarshalByRefObject
    Implements test
    Public Sub _Set(ByVal Data As System.Int16()) Implements test._Set
        Throw New Exception("The method or operation is not implemented.")
    End Sub

    Public Function Set2(ByVal Data As System.Int16()) As System.Int32 Implements test.Set2
        Throw New Exception("The method or operation is not implemented.")
    End Function

End Class
